packages/react/etc/react.api.md

```diff
@@ -4839,6 +4842,7 @@ export interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderI
     rowFieldsAs?: IComponentAs<IDetailsRowFieldsProps>;
     styles?: IStyleFunctionOrObject<IDetailsRowStyleProps, IDetailsRowStyles>;
     theme?: ITheme;
+    totalNestingDepth?: number;
```

Comment:
this facilitates the alignment of the items and defaults to the groupNestingDepth for when `<DetailsRow>` is used for rendering by a `<GroupedList>`